java/tsfile/src/main/java/org/apache/tsfile/read/filter/operator/ValueFilterOperators.java

```diff
@@ -794,6 +886,16 @@ public boolean valueSatisfy(Object value) {
       return !candidates.contains(value);
     }
 
+    @Override
+    protected boolean canSkip(Statistics<? extends Serializable> statistics) {
+      throw new NotImplementedException();
```

Comment:
return false? why throw exception here?

Reply:
Can IoTDB push down NOT IN operator for now?

Reply:
Anyway, return false won't be wrong. Fixed.